matrixio-core.c

```diff
@@ -150,15 +183,16 @@ static int matrixio_init(struct matrixio *matrixio,
 		return ret;
 	}
 
-	if(pdata && pdata->platform_init) {
+	if (pdata && pdata->platform_init) {
 		ret = pdata->platform_init(matrixio->dev);
-		if(ret != 0) {
+		if (ret != 0) {
```

Comment:
You've been doing "if(ret)" in the code. It's better to be consistent. It's OK like this though.